### PR TITLE
navbar update

### PR DESCRIPTION
**Pull request by zldzksk1**
navbar update

### CODE REVIEW

File: book_webframe/src/components/Navigation.js [hunk 1]
```diff
@@ -1,35 +1,45 @@
...
+                <Nav className="mr-auto" >
+                    <Nav.Link href="/Home">HOME</Nav.Link>
+                    <NavDropdown title="BOOK SWAP" id="collasible-nav-dropdown"
+                    show={show}
+                    onMouseEnter={showDropdown}
+                    onMouseLeave={hideDropdown}
+                    >
+                        <NavDropdown.Item href="/Bookpost">Post Books</NavDropdown.Item>
+                        <NavDropdown.Item href="/Booklist">Search Books</NavDropdown.Item>
+                        <NavDropdown.Item href="/ManageRequests">Mange Requests</NavDropdown.Item>
```
Comment (ravishruthi) [inline book_webframe/src/components/Navigation.js:29]: spelling error - Mange to Manage

File: book_webframe/src/components/Navigation.js [hunk 2]
```diff
@@ -1,35 +1,45 @@
...
+    }
+    
+        return(
+            <Navbar collapseOnSelect expand="lg" bg="dark" variant="dark">
+            <Navbar.Brand href="">BINDER</Navbar.Brand>
+            <Navbar.Toggle aria-controls="responsive-navbar-nav" />
+            <Navbar.Collapse id="responsive-navbar-nav">
+                <Nav className="mr-auto" >
+                    <Nav.Link href="/Home">HOME</Nav.Link>
+                    <NavDropdown title="BOOK SWAP" id="collasible-nav-dropdown"
```
Comment (ravishruthi) [inline book_webframe/src/components/Navigation.js:22]: not high priority 
spelling error - collasible to collapsible
Just incase anyone uses id, it doesn't populate like that everywhere?

Comment (zldzksk1) [reply book_webframe/src/components/Navigation.js:22]: Thanks, I will fix the typo, and the ID is from the react-bootstrap

File: book_webframe/src/components/Navigation.js (hunk 1 above)
Comment (zldzksk1) [reply book_webframe/src/components/Navigation.js:29]: Thanks, I will make a change accordingly

**Review by pamelayin — APPROVED** [review]
Looks good, should be okay to push after typo fixes

**Review by zldzksk1 — COMMENTED** [review]
I revised the typo mange -> manage. And collasbile is not typo, it is from react-bootstrap

**Pull request merged**